chapters/planning-data-work.tex

```diff
@@ -483,26 +456,33 @@ \subsection{Documenting and organizing code}
 They all come from the principle that code is an output by itself,
 not just a means to an end,
 and should be written thinking of how easy it will be for someone to read it later.
+At the end of this section, we include a template for a Master Script in Stata, 
+to provide a concrete example of the required elements and structure.
+Throughout this section, we refer to lines of the example .do file 
+to give concrete examples of the required code elements, organization and structure.  
 
 Code documentation is one of the main factors that contribute to readability.
 Start by adding a code header to every file.
 A code header is a long \textbf{comment}\sidenote{
   \textbf{Comments:} Code components that have no function,
   but describe in plain language what the code is supposed to do.
 }
-that details the functionality of the entire script.
+that details the functionality of the entire script;
+refer to lines 5-10 in the example .do file. 
```

Comment:
```suggestion
refer to lines 5-10 in the example do file. 
```

Reply:
Stata write them as `do-files` https://www.stata.com/manuals13/u16.pdf

Reply:
Agree on how to write do-file